### ISSUE TITLE
Export SDL during build?

### ISSUE THREAD

**Issue by v-morlock** [async-graphql/async-graphql#1255]
Is there any way to export the schema SDL during build, e.g. using a build script?

**Comment by xoac** [comment]
Did you tried this?: https://async-graphql.github.io/async-graphql/en/sdl_export.html

**Comment by v-morlock** [comment]
Yes that's working fine, I was wondering if there is any way to export a schema file as part of the regular build process started by `cargo build`

**Comment by xoac** [comment]
You probably need 2 crates then. First library and second 2 final binary. Then you can specify library in `[build-dependencies]` and have access to the export SDL for binary. 

Or look for solution how to access own crate/library from `build.rs` as this is what you are looking for
